terraform/aws-public/mesos-slaves.tf

```diff
@@ -8,7 +8,7 @@ resource "atlas_artifact" "mesos-slave" {
 /* Mesos slave instances */
```

Comment:
As enxebre mentioned the interpolation is not supported by terraform. I tested it with terraform version 0.6.1 (the newest one). So the code:
```
varname = "atlas_artifact.mesos-master.metadata_full.region-${var.region}”

ami = ${varname}
```
and my version mention in the [terraform ticket](https://github.com/hashicorp/terraform/issues/732): 

```
ami = "${lookup(atlas_artifact.mesos-master.metadata_full, concat("region-", var.region))}"
```
don't work.